src/ring_theory/polynomial/cyclotomic/basic.lean

```diff
@@ -487,7 +501,7 @@ begin
       cyclotomic'_eq_X_pow_sub_one_div hpos hz, finset.prod_congr (refl k.proper_divisors) h]
 end
 
-/-- Any `n`-th primitive root of unity is a root of `cyclotomic n ℤ`.-/
+/-- Any `n`-th primitive root of unity is a root of `cyclotomic n K`.-/
```

Comment:
Since the proof of this lemma is just one line, and it is needed only once, in the proof of `is_root_cyclotomic_iff'` I suggest to remove it. At the end of the day `is_root_cyclotomic_iff` is the only result that matters.

Reply:
we discussed this in #10422 - I just tagged you